l10n_co_geo/data/res_bank.xml

```diff
@@ -0,0 +1,864 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+  <data noupdate="1">
+    <!-- <record model="res.bank" id="bank0">
```

Comment:
Well, if they are actually valid, I would let them loaded. If not, then include a comment saying so for anyone to come to the code knows your reasons.